src/main/java/com/epam/preprod/sirenko/util/ConverterToTimestamp.java

```diff
@@ -0,0 +1,39 @@
+package com.epam.preprod.sirenko.util;
+
+import com.epam.preprod.sirenko.ConsoleReader;
+import com.epam.preprod.sirenko.PrintToConsole;
+
+import java.io.IOException;
+import java.sql.Timestamp;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class ConverterToTimestamp {
```

Comment:
Its not just converter.Here we have input,validation and convertion.Maybe we divide to separate classes